_includes/misc/supported_versions.md

```diff
@@ -4,6 +4,7 @@ Supported versions of {{site.productname}}:
 
 | Version |   Release Date    |  End of Support   |
 |:-------:|:-----------------:|:-----------------:|
+|   5.9   | TBD               |        TBD        |
 |   5.8   | May 19, 2021      |        TBD        |
```

Comment:
Don't forget to add the end date for 5.8 here.

Reply:
see https://github.com/tinymce/tinymce-docs/pull/2096